src/views/ApartmentApp/components/StudentApplication/components/OffCampusSection/components/ProgramListItem/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Divider,
+  FormControl,
+  Grid,
+  Input,
+  InputLabel,
+  ListItem,
+  ListItemText,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+const ProgramListItem = ({
```

Comment:
Please use a more consistent naming scheme for these components, such as `OffCampusList` and `OffCampusListItem`, to match both the code style of the 360 project and to match the official terminology of the ApartApp project